src/components/Book.js

```diff
@@ -16,7 +17,44 @@ class Book extends React.Component {
       note: PropTypes.string,
       purchased: PropTypes.bool,
       prioritize: PropTypes.bool
-    }).isRequired
+    }).isRequired,
+    adminMode: PropTypes.bool.isRequired,
+    deleteBook: PropTypes.func.isRequired,
+    isbn: PropTypes.string.isRequired,
+    loadBook: PropTypes.func.isRequired
+  };
+
+  handleEdit = () => {
+    this.props.loadBook(this.props.isbn);
```

Comment:
I'll probably understand this later, but why is `handleEdit`, which I assume is a Command, doing something like `loadBook` which feels like a Query? Maybe `loadBook` is an api call or something. Time to read on!

Reply:
handleEdit is the function attached to the "Edit" button on the Book. However, the name is misleading, because what it _really_ does is load the book data into the Edit Form. Hence, `loadBook`. I agree this is confusing. I renamed most of these methods repeatedly during development. It's confusing because to the user, they want to Edit the book, so Edit is the right word, but to the developer, there's no Editing happening.

🤷‍♂️